integrationExamples/testBidder/testBidderVideoExample.html

```diff
@@ -0,0 +1,89 @@
+<html>
+    <head>
+        <title>Prebid Test Bidder Example</title>
+        <script src="https://cdn.jwplayer.com/libraries/l5MchIxB.js"></script>
+        <script src="../../build/dev/prebid.js" async></script>
+        <script>
+            var pbjs = pbjs || {};
+            pbjs.que = pbjs.que || [];
+
+            const adUnitCode = 'adUnit-0000';
+
+            const adUnits = [{
+                mediaTypes: {
+                    video: {
+                        playerSize: [640, 360]
+                    }
+                },
+                code: adUnitCode,
+                bids: [
+                    {bidder: 'testBidder', params: {}}
+                ]
+            }]
+
+            pbjs.que.push(function () {
+
+                /**
+                 * BID RESPONSE SIMULATION SECTION START
+                 *
+                 * This section handles simulating a bidder
+                 * that will always respond with bid responses.
+                 *
+                 * This part should not be present in production.
+                 */
+                pbjs.registerBidAdapter(null, 'testBidder', {
+                    supportedMediaTypes: ['banner', 'video', 'native'],
+                    isBidRequestValid: () => true
+                });
+
+                pbjs.setConfig({
+                    debugging: {
+                        enabled: true,
+                        intercept: [
+                            {
+                                when: {
+                                    bidder: 'testBidder',
+                                },
+                                then: {
+                                    creativeId: 'testCreativeId',
+                                }
+                            }
+                        ]
+                    }
+                });
+                /**
+                 * BID RESPONSE SIMULATION SECTION END
+                 */
+
+                pbjs.addAdUnits(adUnits);
+                pbjs.requestBids({
+                    adUnitCodes: [adUnitCode],
+                    bidsBackHandler: function() {
+                        const bids = pbjs.getHighestCpmBids(adUnitCode);
+                        const winningBid = bids[0];
+                        jwplayer("player").setup({
```

Comment:
could we put this in a renderer? https://docs.prebid.org/dev-docs/show-outstream-video-ads.html

if `bid.renderer.render`  can load jwplayer + do what you have here, it would/should work without additional setup.